tests/Issues/IssueDowngradeThrowableType/config/configured_rule.php

```diff
@@ -3,10 +3,13 @@
 declare(strict_types=1);
 
 use Rector\Config\RectorConfig;
+use Rector\Core\ValueObject\PhpVersion;
 use Rector\DowngradePhp70\Rector\FunctionLike\DowngradeThrowableTypeDeclarationRector;
 use Rector\DowngradePhp74\Rector\ArrowFunction\ArrowFunctionToAnonymousFunctionRector;
 
 return static function (RectorConfig $rectorConfig): void {
     $rectorConfig->rule(ArrowFunctionToAnonymousFunctionRector::class);
     $rectorConfig->rule(DowngradeThrowableTypeDeclarationRector::class);
+
+    $rectorConfig->phpVersion(PhpVersion::PHP_70);
```

Comment:
is it actually needed? It previously work even without it

Reply:
It is, because of https://github.com/rectorphp/rector-downgrade-php/pull/16

Before the union nullable was change to pure docblock, even if the PHP version was supporting it:

```diff
-public function run(): int|null
+/**
+ * @return int|null
+ */ 
+public function run()
```

Now the nullable value is used with better type control:

```diff
-public function run(): int|null
+public function run()
```

Reply:
Ok 👍